pkgs/by-name/pr/prometheus-frr-exporter/package.nix

```
@@ -0,0 +1,34 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "prometheus-frr-exporter";
+  version = "1.2.0";
```

nit: https://nix.dev/guides/best-practices.html#recursive-attribute-set-rec
```suggestion
let
  version = "1.2.0";
in
buildGoModule {
  pname = "prometheus-frr-exporter";
  inherit version;
```